src/main/java/org/isf/utils/db/DbJpaUtil.java

```diff
@@ -109,12 +92,10 @@ public EntityManager getEntityManager() {
      * Method to persist an object
      * @throws OHException 
      */
-    public void persist(
-    		Object entity) throws OHException
-    {    	
-    	try {
-		    LOGGER.debug("Persist: {}", entity);
-    		entityManager.persist(entity);  
+	public void persist(Object entity) throws OHException {
+		try {
+			LOGGER.debug("Persist: {}", entity);
+			entityManager.persist(entity);
 		} catch (EntityExistsException entityExistsException) {
```

Comment:
maybe rename to `EntityExistsException eee` for more readability

Reply:
I disagree with this suggestion (and the next two); I believe the longer names makes things more clear.

BTW, I added those names on May 31, 2021 as part of OP-524.